subcommittees/mentoring/roles/mentors.md

```diff
@@ -0,0 +1,15 @@
+# Mentors
+
+## Responsibilities
+
+
+## Time commitment
+
+
+## Requirements
+
+
+## To Volunteer
```

Comment:
The join the mentoring program, email the Data Carpentry Associate Director at ebecker@datacarpentry.org.